llvm/test/tools/gold/X86/time-trace.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: llvm-as %s -o %t.o
+
+; RUN: %gold -plugin %llvmshlibdir/LLVMgold%shlibext \
+; RUN:    -m elf_x86_64 --plugin-opt=time-trace=%t2.json \
+; RUN:    -shared %t.o -o %t3.s
```

Comment:
If the output is unused, prefer `-o /dev/null`; otherwise `.so` for shared objects.